src/main/java/pl/kubiczak/orrrg/test/orrrgtest/Hello.java

```diff
@@ -10,4 +10,14 @@ String getMessage(String name) {
 
     return "Hello " + name + "!";
   }
+
+  // this is for sonar to report new issues
+  // on a branch travis run build without Sonarqube Check: https://travis-ci.org/orrrg/test/builds/213231858
+  // but there are regexps for `branches` within addons section (.travis.yml)
+  private void notUsedMethod(){

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "notUsedMethod" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)